src/vs/workbench/services/files/node/fileService.ts

```diff
@@ -1235,7 +1233,10 @@ export class StatResolver {
 					},
 
 					function stat(this: any): void {
-						fs.stat(fileResource.fsPath, this);
+						extfs.statLink(fileResource.fsPath, (error: Error, statAndIsLink) => {
```

Comment:
Can be nicer:

```typescript
extfs.statLink(fileResource.fsPath, (error: Error, { stat, isSymbolicLink }) => {
	isSymbolicLink = isSymbolicLink;
	this(null, stat);
});
```

Reply:
Did not do this due to name clash of the isSymoblicLink variable a couple of lines up. Decided to leave like it is.